packages/experiments/src/components/Button/BaseButton.tsx

```diff
@@ -8,20 +8,30 @@ const baseTokens: IButtonComponent['tokens'] = {
   backgroundColorHovered: undefined,
   backgroundColorPressed: undefined,
   borderColor: undefined,
+  borderColorFocused: undefined,
   borderColorHovered: undefined,
   borderColorPressed: undefined,
   borderRadius: undefined,
-  borderWidth: 2,
+  borderWidth: undefined,
   color: undefined,
   colorHovered: undefined,
   colorPressed: undefined,
-  contentPadding: 0,
-  cursor: 'default',
+  contentPadding: '1px 6px',
+  contentPaddingFocused: undefined,
+  cursor: undefined,
+  height: undefined,
+  iconColor: undefined,
+  iconColorHovered: undefined,
+  iconColorPressed: undefined,
+  iconSize: undefined,
+  iconWeight: undefined,
+  lineHeight: undefined,
   minHeight: undefined,
   minWidth: undefined,
-  textFamily: 'Arial',
-  textSize: 13.3333,
-  textWeight: 400
+  textFamily: undefined,
+  textSize: 13.333,
+  textWeight: undefined,
+  width: undefined
```

Comment:
Just leaving a note per our verbal discussion, it'd be better to have these components tiered in a way where one doesn't have to erase styling from the other.

Reply:
Added a step for this in #6134.